association.go

```diff
@@ -982,29 +982,30 @@ func (a *Association) handleHeartbeat(c *chunkHeartbeat) []*packet {
 func (a *Association) handleCookieEcho(c *chunkCookieEcho) []*packet {
 	state := a.getState()
 	a.log.Debugf("[%s] COOKIE-ECHO received in state '%s'", a.name, getAssociationStateString(state))
-	if state != closed && state != cookieWait && state != cookieEchoed {
-		return nil
-	}
+	if state != established {
+		if state != closed && state != cookieWait && state != cookieEchoed {
+			return nil
+		}
```

Comment:
```suggestion
	switch state {
	default:
		return nil
	case established:
		break
	case closed, cookieWait, cookieEchoed:
```
may be bit readable.

Reply:
BTW, is it needed to handle cookie echo after being closed?

Reply:
Thanks @at-wat really appreciated. Good point.  Let me check the RFC...

Reply:
Yes, we need to handle Cookie Echo chunk in closed state. See [this diagram](https://github.com/pion/webrtc/issues/1270#issuecomment-650857414)